docs/unified-test-documentation/dasharo-compatibility/362-coreboot-customizer.md

```diff
@@ -0,0 +1,93 @@
+# Dasharo Compatibility: DMI editing tool
+
+## DMI001.001 Change the UUID
+
+**Test description**
+
+The coreboot_customizer tool is a part of the Dasharo Tools Suite. It allows to
+customize DMI information in a Dasharo .rom image. In this test case we will
+try to change the UUID.
+
+**Test configuration data**
+
+1. `FIRMWARE` = Dasharo
+
+**Test setup**
+
+1. Proceed with the
+    [Generic test setup: firmware](../generic-test-setup.md#firmware).
+1. Download [coreboot_customizer](https://github.com/Dasharo/meta-dts/blob/main/meta-dts-distro/recipes-dts/dts/dts/coreboot_customizer)
+1. Compile cbfstool as explained [here](https://github.com/Dasharo/docs/blob/9ab374589a32a17657c9f0c30c37da6637eea462/docs/guides/image-customization.md#compiling-cbfstool)
```

Comment:
This is optional. As mentioned in the documentation:

```
Additionally the script makes use of [coreboot's cbfstool](https://github.com/coreboot/coreboot/tree/master/util/cbfstool) to modify the images, so it is required to be available to the script. The script will try to find the `cbfstool`` installed in the system. If not found, then it searches for cbfstool in the current directory. If not found, then it tries to download the cbfstool from [3mdeb's FTP](https://dl.3mdeb.com/open-source-firmware/utilities/cbfstool) (compiled using instructions in [Compiling cbfstool](https://docs.dasharo.com/guides/image-customization/#compiling-cbfstool)). If that fails, the script will exit with an error.
```

So it should download the cbfstool if not found.

Reply:
Fixed in [recent commit](https://github.com/Dasharo/docs/commit/2f2c60da5365e1c65115c59cf2a341885221995f)